### PR TITLE
Solution/find the neighbour

### PR DESCRIPTION
**Pull request by ederalab**

### CODE REVIEW

**Comment by ederalab** [comment]
Reviewers:
@micheleriva 
@Gsantomaggio

File: challenges/easy/clean-up-after-your-dog/solutions/valentinadagata/main.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+def cleanUp(garden, bags, cap):
```
Comment (micheleriva) [inline challenges/easy/clean-up-after-your-dog/solutions/valentinadagata/main.py:1]: This file refers to another challenge, please delete it before merging the PR

File: challenges/intermediate/find-the-neighbour/solutions/valentinadagata/main.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+def find_neighbour(cube, query):
+    # delete all characters that are not letters
+    regex = re.compile('[^a-zA-Z]')
+    newcube = regex.sub(' ', cube)
+    # transform the string to list
+    newcube = newcube.split()
+    # count how many rows and cols there are (so that it could be scalable)
+    rows_cols = int(sqrt(len(newcube)))
+    # transform the list into a matrix
+    matrix = [newcube[x:x + rows_cols] for x in range(0, len(newcube), rows_cols)]
```
Comment (micheleriva) [inline challenges/intermediate/find-the-neighbour/solutions/valentinadagata/main.py:13]: nice usage of list comprehension, I like it

**Review by micheleriva — CHANGES_REQUESTED** [review]
The solution is awesome, but please remove the other challenge file!

**Comment by micheleriva** [comment]
@ederalab it looks like you have a conflict on the `challenges/easy/clean-up-after-your-dog/solutions/valentinadagata/main.py` file. Could you please resolve this so that I can merge your PR? Thanks!

**Comment by ederalab** [comment]
I updated the file and I hope the new one has no conflicts. I also deleted  the "clean-up-after-your-dog" files from the local branch

**Pull request closed without merging**